curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter.md

```diff
@@ -55,13 +55,13 @@ The function will return the correct conversion if the supplied problems are pro
 
 - Situations that will return an error:
   - If there are **too many problems** supplied to the function. The limit is **five**, anything more will return:
-    `Error: Too many problems.`
+    `"Error: Too many problems."`
```

Comment:
```suggestion
    `'Error: Too many problems.'`
```
for python the style choice is to use single quotes unless otherwise necessary

Reply:
Hey Thanks for the feedback, i looked over the messages on the main issue before starting this part, my understanding was to use `"` by default for consistency, the strings that do have a quotation have been `"` and not ` ' `.

> So, whilst that could be an exception for the Python curriculum, I would prefer the default being `"`.[^see]
[^see](https://github.com/freeCodeCamp/freeCodeCamp/issues/51036#issuecomment-1914733986)

let me know your thoughts and would be happy to change it if i am misunderstood.

Reply:
this is the Python curriculum, as a stylistic choice the quotes to use are single quotes

Reply:
changes applied in commit #64ffe79